.github/README.md

```diff
@@ -64,44 +64,78 @@ Javalin.create { config ->
+pemFromClasspath("certName.pem","keyName.pem");                       // load from the given paths in the classpath
+pemFromClasspath("certName.pem","keyName.pem","keyPassword");         // load from the given paths in the classpath with the given key password
+pemFromInputStream(certInputStream,keyInputStream);                   // load from the given input streams
+pemFromInputStream(certInputStream,keyInputStream,"keyPassword");     // load from the given input streams with the given key password
+pemFromString(certString,keyString);                                  // load from the given strings
+pemFromString(certString,keyString,"keyPassword");                    // load from the given strings with the given key password
 
 // Keystore loading options (PKCS#12/JKS) (mutually exclusive)
 keystoreFromPath("/path/to/keystore.jks","keystorePassword");         // load the keystore from the given path
-keystoreFromClasspath("keyStoreName.p12","keystorePassword");         // load the keystore from the given path in the classpath.
-keystoreFromInputStream(keystoreInputStream,"keystorePassword");      // load the keystore from the given input stream.
+keystoreFromClasspath("keyStoreName.p12","keystorePassword");         // load the keystore from the given path in the classpath
+keystoreFromInputStream(keystoreInputStream,"keystorePassword");      // load the keystore from the given input stream
 
 // Advanced options
-configConnectors = null;                                              // Consumer to configure the connectors.
-securityProvider = Conscrypt.newProvider();                           // Set the security provider to use.
+configConnectors(Consumer<ServerConnector>);                          // Set a Consumer to configure the connectors
+securityProvider = null;                                              // Use a custom security provider
+withTrustConfig(Consumer<TrustConfig>);                               // Set the trust configuration, explained below. (by default all clients are trusted)
 ```
 
-### Hot reloading
-Certificate reloading is supported by default, if you want to replace the certificate you can simply call `SSLPlugin#reload` with the new certificate:
+#### Trust Configuration
 
+If you want to verify the client certificates (such as mTLS) you can set the trust configuration using the `TrustConfig` class.
+In contrast to the identity configuration, you can load multiple certificates from different sources.
+
+By adding a `TrustConfig` to the `SSLPlugin` you will enable client certificate verification.
 ```java
+config.plugins.register(new SSLPlugin(ssl->{
+    ssl.pemFromPath("/path/to/cert.pem","/path/to/key.pem"); // Load our identity data
+    // Load the client/CA certificate(s)
+    ssl.withTrustConfig(trust->{
+        trust.certificateFromPath("/path/to/clientCert.pem");
+        trust.certificateFromClasspath("rootCA.pem");
+    });
+}));
```

Comment:
```java
config.plugins.register(new SSLPlugin(ssl->{
    ssl.pemFromPath("/path/to/cert.pem","/path/to/key.pem"); // Load our identity data
    // Load the client/CA certificate(s)
    ssl.withTrustConfig(trust->{
        trust.certificateFromPath("/path/to/clientCert.pem");
        trust.certificateFromClasspath("rootCA.pem");
    });
}));
```

`ssl.pemFromPath` sets the key material of the server but to set the trust material it needs to be configured one level deeper like `ssl.withTrustConfig(trust -> trust.certificateFromPath` but wouldn't it make more sense to have something like this:

```java
config.plugins.register(new SSLPlugin(ssl->{
    ssl.withIdentityConfig(identity->{
        identity.pemFromPath("/path/to/cert.pem","/path/to/key.pem"); // Load our identity data
    });

    // Load the client/CA certificate(s)
    ssl.withTrustConfig(trust->{
        trust.certificateFromPath("/path/to/clientCert.pem");
        trust.certificateFromClasspath("rootCA.pem");
    });
}));
```

I just noticed this and I wanted to share my thoughts but I am not familar with the code base and usage of Javalin at all, so not quite sure whether this feedback is valid but that is up to you to decide 😄

Reply:
It's purposely done for novice users, which is many of Javalin's. I don't want people confusing both, I expect most users (95%+) to only use it for identity information. Also the _Identity_/_Trust_ nomenclature is pretty Java specific and I'd like to avoid it as much as posible. If you check Javalin's homepage you'll see one of the goals: 
![image](https://user-images.githubusercontent.com/23284823/212569467-ce719396-c61e-4b07-bfb0-2ef69ff95a92.png)
Think of this plugin of a VERY simple way of configuring SSL for basically dummies, any more advanced users should configure their own Jetty connectors, using you library directly for example.

Reply:
Ah okay, that makes sense. Thank you for the explanation